### PR TITLE
feat: hub make decode work with -d or --decode

### PR DESCRIPTION
**Pull request by cloud-pharaoh**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/hub/scripts/gke_hub_registration.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ PROJECT_ID=$5
 tmp_file=$(mktemp)
 # shellcheck disable=SC2064
 trap "rm -rf $tmp_file" EXIT
-echo "${SERVICE_ACCOUNT_KEY}" | base64 --decode > "$tmp_file"
+base64 --help | grep "\--decode" && B64_ARG="--decode" || B64_ARG="-d"
```
Comment (bharathkkb) [inline modules/hub/scripts/gke_hub_registration.sh:33]: @cloud-pharaoh base64 should accept both `-d` and ` --decode` right?

Comment (cloud-pharaoh) [reply modules/hub/scripts/gke_hub_registration.sh:33]: Depending on OS.
On my mac it does. On my alpine based runner it only supports -d so it broke when I tried to use it.

I saw this pattern in kubectl_wrapper.sh script for multiple modules and it works now :)

Comment (bharathkkb) [reply modules/hub/scripts/gke_hub_registration.sh:33]: @cloud-pharaoh makes sense, could you rebase this. For some reason the github UI does not show the update branch button for me.

Comment (cloud-pharaoh) [reply modules/hub/scripts/gke_hub_registration.sh:33]: Done 👍

**Review by bharathkkb — APPROVED** [review]
LGTM

**Pull request merged**